docs/user-guide/performance/parameters.md

```diff
@@ -15,11 +15,19 @@ helm upgrade --install \
 --set http.url=https://httpbin.org/get
 ```
 
-All the parameters of the performance test (including of all included tasks) are optional unless otherwise documented.
+All the parameters of the performance test (including of all included tasks) are optional unless otherwise documented. The task specific parameters are documented in the task documentation.
 
-## Parameters
+Currently available tasks are:
 
-Global performance test parameters are described here. Task specific parameters are documented in each task description.
+- [`http`](tasks/http.md) - generate synthetic load and capture performance metrics for HTTP endpoints
+- [`grpc`](tasks/grpc.md) - generate synthetic load and capture performance metrics for gRPC methods
+- [`ready`](tasks/ready.md) - test readiness of an object
```

Comment:
```suggestion
- [`ready`](tasks/ready.md) - checks readiness of an object
```

I think it might make sense to avoid using the word "test" as much as possible here